proj/cmake/modules/findCMakeDirs.cmake

```diff
@@ -6,7 +6,7 @@ macro( findCMakeDirs RESULT_PATHS BASE_PATH SKIP_PATHS )
 			set( shouldSkip FALSE )
 			foreach( skip ${SKIP_PATHS} )
 				if( f MATCHES ".*${skip}" )
-					ci_log_v( "---- [findCMakeDirs] skipping path: ${f}" )
```

Comment:
I just think it's important for users to know which samples are ignored, but that's debatable.

Reply:
Makes sense to me. `CINDER_SKIP_SAMPLES` is user-settable anyway, I added it as a stop-gap soln for building all samples in one go. Hopefully, it won't be needed much after this work gets merged. :)